components/speedreader/speedreader_url_loader.cc

```diff
@@ -29,6 +31,24 @@ namespace {
 
 constexpr uint32_t kReadBufferSize = 32768;
 
+void SaveDistilledData(const GURL& url,
+                       const std::string& data,
+                       const std::string& stylesheet,
+                       const std::string& transformed) {
+#if _DEBUG
```

Comment:
do you really build debug browser?

Reply:
Always

Reply:
imo, we shouldn't use debug since nobody builds it, there are no goma caches, etc